Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -293,7 +292,6 @@
   - PAIN
   - PHYSICS
   - POVERTY
-  - PRIDE
```

Comment:
pride is not only gay pride

Reply:
It was the main point in the issue, that it can lead to sexist outcomes. so i don't see the issue with removing it